Tests/NIOHTTP2Tests/TestUtilities.swift

```diff
@@ -79,8 +79,9 @@ extension XCTestCase {
     func assertDoHandshake(client: EmbeddedChannel, server: EmbeddedChannel,
                            clientSettings: [HTTP2Setting] = nioDefaultSettings, serverSettings: [HTTP2Setting] = nioDefaultSettings,
                            file: StaticString = #file, line: UInt = #line) throws {
-        client.pipeline.fireChannelActive()
-        server.pipeline.fireChannelActive()
+        let socket = try SocketAddress(unixDomainSocketPath: "/fake")
+        _ = try client.connect(to: socket).wait()
+        _ = try server.connect(to: socket).wait()
```

Comment:
This isn't quite right, semantically speaking. Maybe change this to `server.registerAlreadyConfigured0`.

Reply:
this doesn't work since `EmbeddedChannel` doesn't forward `registerAlreadyConfigured0` to `channelcore`. also tried a bunch of combinations of `bind` and `connect`, and this one was the only one that worked without having to modify `swift-nio` as well. 

Would you prefer that I add the proxy method in `EmbeddedChannel` in another PR?

Reply:
Gah, you're right. Ok, let's just put a comment on this to say that it's not right, but is required to activate the embedded channel. We can then update EmbeddedChannel in a separate PR.